examples/reprojection-by-code.js

```diff
@@ -3,6 +3,7 @@ import _ol_View_ from '../src/ol/View.js';
 import * as _ol_extent_ from '../src/ol/extent.js';
 import _ol_layer_Tile_ from '../src/ol/layer/Tile.js';
 import {get as getProjection, getTransform} from '../src/ol/proj.js';
+import {register} from '../src/ol/proj/proj4.js';
```

Comment:
Should this example also `import proj4 from 'proj4'` instead of relying on the global?